### DISCUSSION TITLE
Dịch inertia, silhouette score

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#70] [Terminology] [answered]
Hai từ này đều là metric trong bài toán clustering:
* `inertia`: `quán tính`
* `silhouette`: giữ nguyên

cc @mlbvn/handson-ml2

**Comment by quangnhat185** [comment]
Từ `silhouette` khó nhằn nhỉ, có thể dịch là `tính phản chiếu`  ko?

**Reply by lkhphuc** [reply]
`Silhouette` tại sao không dịch theo đúng nghĩa gốc là `hình bóng` nhỉ? Nó không đúng trong ngữ cảnh này hả?

**Reply by minhduc0711** [reply]
mình chưa hình dung rõ ý nghĩa của tên gọi này.. https://en.wikipedia.org/wiki/Silhouette_(clustering)

**Comment by minhduc0711** [comment]
chốt tạm nhé mọi người
* `inertia`: `quán tính`
* `silhouette`: giữ nguyên

**Reply by rootonchair** [reply]
Em nghĩ mình có thể giữ nguyên chứ `inertia` không ạ. Vì em không thấy định nghĩa chính thống cho từ này nên không biết nó có quan hệ gì với `quán tính` trong vật lý không

**Reply by ngcthuong** [reply]
inertia không có định nghĩa chính thống ah? Mấy từ này thử mang lên forum/group facebook hỏi xem. Mình đọc qua nghe cũng lạ, vì cái tên có vẻ ko phản ánh ý nghĩa của nó.

**Reply by minhduc0711** [reply]
Thế đổi lại thành từ gốc vậy, mình cũng chưa tìm được lời giải thích cho cái tên này.

**Comment by minhduc0711** [comment, marked as answer]
Mình lại giữ nguyên cả 2 từ `inertia` và `silhouette` nhé.
